filters/include/pcl/filters/impl/crop_hull.hpp

```diff
@@ -135,46 +122,21 @@ pcl::CropHull<PointT>::applyFilter2D (PointCloud &output)
          ))
       {
         if (crop_outside_)
-          output.push_back ((*input_)[(*indices_)[index]]);
-        // once a point has tested +ve for being inside one polygon, we can
```

Comment:
Would be great if you could save these other comments as well

Reply:
returned back lost comments